less/misc.less

```diff
@@ -53,6 +53,9 @@ input, textarea {
 	visibility: hidden;
 	top: 0;
 	float: left;
+	.selectlist {
+		display: block !important;
```

Comment:
If "sizer" is good enough for the js variable name, why are we deviating from that in the CSS? Now it is inconsistent and _even more confusing_.